### DISCUSSION TITLE
Modification of Nussel number in simulation under low pressure environment in FDS source code

### DISCUSSION THREAD

**Discussion by Eggcellent927** [firemodels/fds#13444] [Q&A]
Dear all,
I am currently working on simulating the combustion phenomena of materials in low-pressure environments. I came across an article that mentioned modifying the Nusselt number in the FDS source code as Nu' = Nu(P/P0)^(2/15) to generate a new simulator program for low-pressure applications. I am wondering in which part of the source code this modification should be made. Additionally, I would like to know if this modification is reasonable.
Here is the online preview address of the original literature: https://onlinelibrary.wiley.com/doi/full/10.1002/fam.2768. The specific modification can be found in Equation (10).
I sincerely look forward to your replies and assistance. Thank you very much for your time and expertise.
[A new approach based on the modified FDS to numerically simulate cardboard box combustion under low pressure.pdf](https://github.com/user-attachments/files/17055221/A.new.approach.based.on.the.modified.FDS.to.numerically.simulate.cardboard.box.combustion.under.low.pressure.pdf)

**Comment by drjfloyd** [comment]
This is not work performed by the FDS development team. If you want to know exactly what was done, your best bet would be to contact the authors of the paper.  I assume they changed the Nu calculation in the subroutines that compute the convective heat transfer coefficient.

It should be noted that paper only shows results of the modified FDS. It isn't shown what amount of improvement, if any, is actually seen.

**Reply by Eggcellent927** [reply]
Thank you for your prompt reply, Dr. Floyd. I will try to contact the authors of the paper for more details.
